Race condition on removal of skupper in TestBasic

#731 introduced the deferred function `Delete(ctx context.Context, t *testing.T)` on `test/integration/acceptance/custom/basic/basic.go`, which removes Skupper from the namespaces with two operations:

    assert.Assert(t, pub1Cluster.VanClient.SiteConfigRemove(ctx))
    assert.Assert(t, pub1Cluster.VanClient.RouterRemove(ctx))

That seems to be causing the following intermittent error on CI test runs:

    --- FAIL: TestBasic/interiors-ingress-nodeport (41.11s)
        basic.go:356: Testing: Connecting two interiors with ingress=nodeport
        basic.go:126: assertion failed: error is not nil: Skupper not installed in 'private-basic-1': deployments.apps "skupper-router" not found

The error has been seen on both policy and non-policy testing (ie, with and without the Policy CRD installed).

--- a/test/integration/acceptance/custom/basic/basic.go
+++ b/test/integration/acceptance/custom/basic/basic.go
@@ -121,10 +121,12 @@ func (r *BasicTestRunner) Delete(ctx context.Context, t *testing.T) {
 	pub1Cluster, _ := r.GetPublicContext(1)
 	prv1Cluster, _ := r.GetPrivateContext(1)
 	if err := pub1Cluster.VanClient.SiteConfigRemove(ctx); err != nil {
-		_ = pub1Cluster.VanClient.RouterRemove(ctx)
+		t.Logf("error removing site config: %v", err)
+		t.Logf("removing router - err: %v", pub1Cluster.VanClient.RouterRemove(ctx))
 	}
 	if err := prv1Cluster.VanClient.SiteConfigRemove(ctx); err != nil {
-		_ = prv1Cluster.VanClient.RouterRemove(ctx)
+		t.Logf("error removing site config: %v", err)
+		t.Logf("removing router - err: %v", prv1Cluster.VanClient.RouterRemove(ctx))
 	}
 	waitNoPods := func(componentSelector string, cluster *base.ClusterContext) error {
 		return utils.RetryWithContext(ctx, time.Second, func() (bool, error) {

--- a/test/integration/acceptance/custom/basic/basic.go
+++ b/test/integration/acceptance/custom/basic/basic.go
@@ -120,10 +120,12 @@ func (r *BasicTestRunner) Delete(ctx context.Context, t *testing.T) {
 	defer cn()
 	pub1Cluster, _ := r.GetPublicContext(1)
 	prv1Cluster, _ := r.GetPrivateContext(1)
-	assert.Assert(t, pub1Cluster.VanClient.SiteConfigRemove(ctx))
-	assert.Assert(t, pub1Cluster.VanClient.RouterRemove(ctx))
-	assert.Assert(t, prv1Cluster.VanClient.SiteConfigRemove(ctx))
-	assert.Assert(t, prv1Cluster.VanClient.RouterRemove(ctx))
+	if err := pub1Cluster.VanClient.SiteConfigRemove(ctx); err != nil {
+		_ = pub1Cluster.VanClient.RouterRemove(ctx)
+	}
+	if err := prv1Cluster.VanClient.SiteConfigRemove(ctx); err != nil {
+		_ = prv1Cluster.VanClient.RouterRemove(ctx)
+	}
 	waitNoPods := func(componentSelector string, cluster *base.ClusterContext) error {
 		return utils.RetryWithContext(ctx, time.Second, func() (bool, error) {
 			pods, _ := kube.GetPods(componentSelector, cluster.Namespace, cluster.VanClient.KubeClient)
@@ -149,8 +151,8 @@ func (r *BasicTestRunner) TearDown(ctx context.Context) {
 		log.Warn(errMsg)
 	}
 
-	pub.DeleteNamespace()
-	priv.DeleteNamespace()
+	_ = pub.DeleteNamespace()
+	_ = priv.DeleteNamespace()
 }
 
 func (r *BasicTestRunner) Run(ctx context.Context, t *testing.T) {

